pkg/memtier/prompt.go

```diff
@@ -582,7 +582,7 @@ func (p *Prompt) cmdPages(args []string) CommandStatus {
 		p.output(`LOCKED        %d
 ERROR         %d
 REFERENCED    %d
-UPTODATE      %d
+UP-TO-DATE    %d
```

Comment:
Let's follow strictly kernel page flag names. They do not have hyphens.

https://www.kernel.org/doc/html/latest/admin-guide/mm/pagemap.html?highlight=UPTODATE